### PR TITLE
[Workers] New Python Workers Example

### PR DESCRIPTION
**Pull request by megaconfidence**
Adds new example for:
- [CORS header proxy](https://developers.cloudflare.com/workers/examples/cors-header-proxy/)
- [Country code redirect](https://developers.cloudflare.com/workers/examples/country-code-redirect/)
- [Custom Domain with Images](https://developers.cloudflare.com/workers/examples/images-workers/)

### CODE REVIEW

**Review by kodster28 — APPROVED** [review]
Content-wise, lgtm

**Comment by megaconfidence** [comment]
Please can you take a look @hoodmane. Thanks!

File: content/workers/examples/cors-header-proxy.md [hunk 1]
```diff
@@ -325,5 +326,115 @@ export default {
...
+
+```py
+from pyodide.ffi import to_js as _to_js
+from js import Response, URL, fetch, Object, Request
+
+def to_js(x):
+    return _to_js(x, dict_converter=Object.fromEntries)
+
+async def on_fetch(request):
+    cors_headers = {
```
Comment (hoodmane) [inline content/workers/examples/cors-header-proxy.md:340]: Could you move most of the body of `on_fetch` to top level definitions?
e.g. `cors_headers`, `api_url`, `raw_html_response`, `handle_request`, `handle_options`? As far as I can tell none of them are closing over per-request data so it's nice to have them in the top level.

File: content/workers/examples/cors-header-proxy.md [hunk 2]
```diff
@@ -325,5 +326,115 @@ export default {
...
+        request.headers["Origin"] = (URL.new(api_url2)).origin
+        print(request.headers)
+        response = await fetch(request)
+        response = Response.new(response.body, response)
+        response.headers["Access-Control-Allow-Origin"] = url.origin
+        response.headers["Vary"] = "Origin"
+        return response
+
+    async def handle_options(request):
+        if request.headers["Origin"] and request.headers["Access-Control-Request-Method"] and request.headers["Access-Control-Request-Headers"]:
```
Comment (hoodmane) [inline content/workers/examples/cors-header-proxy.md]: I think this should be:
```suggestion
        if "Origin" in request.headers and "Access-Control-Request-Method" in request.headers and "Access-Control-Request-Headers" in request.headers:
```

Comment (hoodmane) [reply content/workers/examples/cors-header-proxy.md]: But I think you could also say:
```
if {"Origin", "Access-Control-Request-Method", "Access-Control-Request-Headers"}.issubset(request.headers.keys()):
```

File: content/workers/examples/cors-header-proxy.md (hunk 1 above)
Comment (megaconfidence) [reply content/workers/examples/cors-header-proxy.md:340]: That makes sense, but I'm thinking of leaving it in to keep the examples consistent across languages

**Comment by hoodmane** [comment]
Thanks @megaconfidence!

**Pull request merged**